backend/src/main/java/gov/cdc/usds/simplereport/idp/repository/LiveOktaRepository.java

```diff
@@ -499,24 +413,6 @@ private String generateRoleSuffix(OrganizationRole role) {
   }
 
   private String generateFacilitySuffix(String facilityId) {
-    return ":" + _extractor.FACILITY_ACCESS_MARKER + ":" + facilityId;
-  }
-
-  // Hashes the role env prefix, org external ID, and the roles to migrate to.
-  // NOTE: The Okta UI will still display the source and destination groups for
-  // this rule. This hash approach is in place because Okta secretly limits group
-  // rule names to 50 chars, which is frequently less than the combined length of
-  // the representation of the role env prefix, org external ID, and the roles.
-  private String generateMigrationGroupRuleName(
-      String orgExternalId, Collection<OrganizationRole> roles) {
-    return DigestUtils.sha1Hex(generateGroupOrgPrefix(orgExternalId) + roles.toString());
```

Comment:
Hilariously, I think this gets rid of one of our BUG sonar mentions (or maybe it was Coverity?)—they were very upset that we were using an insecure hashing algorithm.

Reply:
One of the first emails I got about the project was a complaint about this from Coverity.   After taking a look, all I saw was a clever workaround : )  Always nice to get rid of a warning, though.

Reply:
Yeah, it's not like there was an actual problem, or I would have actually done something about it--I just kind of wished we'd used sha256 instead and avoided the annoying email. 😆